### ISSUE TITLE
Demo works; source code kinda works

### ISSUE THREAD

**Issue by rlbaldwin** [scottstensland/websockets-streaming-audio#2]
Hello Scott,

Your demo is exactly what I've been looking for.  That's my good luck.

Symptom: the playback works until two buffers have been loaded into the browser, then it simply loops until I stop it using the STOP button.  I RELOAD and try a different button, same result.  Only the sine wave works since it's only 32K in size.

The online demo works just fine.  I'm using Chrome Version 42.0.2311.90 on Windows 8.1.

Any ideas?

**Comment by scottstensland** [comment]
Yes I have some work to do ... I need to better isolate execution runs ... its on my todo list ... let me know if have any questions

**Comment by BananaHemic** [comment]
I'm having the same issue, unfortunately. Is the source running the online demo the same as the source provided by npm? What's the node version used in the online demo?

**Comment by scottstensland** [comment]
node  0.10.x       ...   using that version it works ...  its what packages.json says and what demo uses

yes I need to get it working on latest node ... have not looked but fix is probably quite simple

**Comment by mariomurrent-softwaresolutions** [comment]
Not yet fixed, right?

**Comment by scottstensland** [comment]
I fixed this over the weekend ... two different root causes ... Web Audio API had an intentional change made to shut off autoPlay and so I just gave a kick in the pants to the audio context prior to first use ... other was just an easy replacement of `new Buffer()` into `new Buffer.alloc()` and its siblings when I bumped nodejs up to the current release 12.14.0  along with bumping up all upstreams including `ws` 

I am not a front end developer so although its ~~working~~ I am actively polishing it so when browser button is hit the server side --> audio rendering is solid ...  as an aside I have a project where I need client side audio streaming and will leverage this project so stay tuned ... same bat time same bat channel
